### ISSUE TITLE
aria-live on a fieldset too verbose in Firefox

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3840]
**Reported by andrewnicols on 2014-02-03 09:03**
In Firefox, when aria-live is set to "polite" on a fieldset, if you type within any input/textarea/select in that fieldset, the input is read out.

If you additionally have the 'Speak typed words' option set, the keys are read out twice.

I've compared results with a number of other browsers and screen readers, and the only combination where this issue is seen is NVDA with Firefox.

For reference, the full list I've compared with is:
- NVDA:
  - IE
  - Chrome
- Jaws:
  - IE
  - Chrome
  - Firefox
- Chromevox
  - Chrome (Windows)
  - Chrome (Mac)
- Mac OS VoiceOver
  - Chrome
  - Firefox
  - Safari

We set aria-live="polite" on the fieldset because we have a toggle which shows/hides the fieldset and need to announce that the region visibility has changed as a result of a user interaction.

**Comment by Adriani90** [comment]
could someone please upload the attachment for reproducing this issue? @ehollig maybe?

**Comment by Adriani90** [comment]
@andrewnicols can you still reproduce it?

**Comment by ehollig** [comment]
@Adriani90 the attachment has been added to https://github.com/nvaccess/nvda/issues/3840#issuecomment-155317136

**Comment by Adriani90** [comment]
I have tested in Firefox 64.0.2 and it is still reproducible with NVDA 2018.4. cc: @jcsteh

**Comment by jcsteh** [comment]
As I noted in https://github.com/nvaccess/nvda/issues/3840#issuecomment-155317141, this is not a bug; it's authoring error. It's also not Firefox specific; I confirmed this happens in Chrome as well.

NVDA may still wish to "work around" this by ignoring live updates on editable text, as I've seen quite a few sites get this wrong and I don't think it's useful under any circumstances. I'd say it's pretty low priority though.

**Comment by Adriani90** [comment]
This is a duplicate of #8442 and will be fixed once #14666 is merged. I am closing this in favor of #8442.
